js/db.js

```diff
@@ -756,6 +756,93 @@ define(["dao", "globals", "lib/bluebird", "lib/davis", "lib/underscore", "util/h
                     playerFeatStore.createIndex("tid", "tid", {unique: false});
                 }());
             }
+            if(event.oldVersion <= 13) {
```

Comment:
Wow, you even did the DB upgrade code! Add a space between if and ( here though (also, you should get ESLint running!).